cmd/tuple/import.go

```diff
@@ -81,6 +82,15 @@ func ImportTuples(
 	return &result, nil
 }
 
+// function to extract the simplified error message.
+func extractErrorMssg(fullErrorMssg string) string {
+	parts := strings.Split(fullErrorMssg, ": ")
```

Comment:
I'm concerned that we're now reducing the error message too much, for example when I write a tuple that already exists then I get `"reason":"invalid write input"` which isn't really an actionable message.

That said, I don't see anything from the Go SDK that would really allow us to improve this as the underlying response from the API isn't available via the SDK unfortunately.

It's not ideal, but if we split on `error message: ` instead then we get two parts and the latter of those two parts seems more suitable `"reason":"cannot write a tuple which already exists: user: 'user:beth', relation: 'viewer', object: 'folder:product-2021': invalid write input"`, what do you think @shruti2522?

Reply:
I am getting the following output, and for this message the function extracts the last relevant part.
```
shruti@fedora:~/cncf/cli$ ./dist/fga tuple write --store-id=01H0H015178Y2V4CX10C2KGHF4 --file example/folder-document-access_tuples.json
{
  "successful":null,
  "failed": [
    {
      "tuple_key": {
        "object":"folder:product",
        "relation":"owner",
        "user":"user:anne"
      },
      "reason":"No authorization models found for store '01H0H015178Y2V4CX10C2KGHF4'"
    },
    {
      "tuple_key": {
        "object":"folder:product-2021",
        "relation":"parent",
        "user":"folder:product"
      },
      "reason":"No authorization models found for store '01H0H015178Y2V4CX10C2KGHF4'"
    },
    {
      "tuple_key": {
        "object":"folder:product-2021",
        "relation":"viewer",
        "user":"user:beth"
      },
      "reason":"No authorization models found for store '01H0H015178Y2V4CX10C2KGHF4'"
    }
  ]
}

```

Reply:
Here's the steps I performed in order to get that: result

1. Started an OpenFGA server
2. Created a store `fga store create --name test`
3. Wrote an example model from this repo `fga model write --file ./example/model.fga --store-id <store-id>`
4.  Wrote the initial tuples using `fga tuple write --file ./example/folder-document-access_tuples.json --store-id <store-id>`
5. Wrote the tuples again using `fga tuple write --file ./example/folder-document-access_tuples.json --store-id <store-id>`

Writing the tuples again in step 5 does return output that can be mapped back to the OpenFGA errors (if a developers knows them), but is missing crucial information returned from the server because the error message itself contains the `:` character.

[Looking at the go-sdk](https://github.com/openfga/go-sdk/blob/main/api_open_fga.go#L4286) we can see that the error message returned from the OpenFGA server is appended to the string following `error message: `, below is the output of `write.Error.Error()` for the above test case and the no model found error that demonstrate this. Given that we don't currently have a way to extract that error message from a go-sdk error, I think that this is the safest way to get the error message without losing any extra information.

```
"Write validation error for POST Write with body {\"code\":\"write_failed_due_to_invalid_input\",\"message\":\"cannot write a tuple which already exists: user: 'user:anne', relation: 'owner', object: 'folder:product': invalid write input\"}\n with error code write_failed_due_to_invalid_input error message: cannot write a tuple which already exists: user: 'user:anne', relation: 'owner', object: 'folder:product': invalid write input"
```

```
"Write validation error for POST Write with body {\"code\":\"latest_authorization_model_not_found\",\"message\":\"No authorization models found for store '01HW300QY4HWVEE48WZ1HAK1JJ'\"}\n with error code latest_authorization_model_not_found error message: No authorization models found for store '01HW300QY4HWVEE48WZ1HAK1JJ'"
```